pyEpiabm/pyEpiabm/sweep/initial_place_sweep.py

```diff
@@ -0,0 +1,130 @@
+                                                                     params)
+
+                if place.place_type.name in schools:  # schools
+                    # Initialise the fixed population
+                    helper.update_place_group(place, person_list=person_list,
+                                              person_weights=weights,
+                                              mean_capacity=mean_cap,
+                                              max_capacity=max_cap,
+                                              group_index=0)
+
+                elif place.place_type.name == "Workplace":  # WORKSPACE
+                    # Fixed population is initialised on first run
+                    # thinking of making sure people don't have more than one
+                    # workplace
+                    for i in range(group_num):
+                        helper.update_place_group(place,
+                                                  person_list=person_list,
+                                                  person_weights=weights,
+                                                  mean_capacity=mean_cap,
+                                                  max_capacity=max_cap,
+                                                  group_index=0)
+                        # Remove added people from list of possible addees.
+                        # Only if there were sufficient people to add.
+                        if i in place.person_groups.keys():
+                            person_list = [person for person in person_list
+                                           if person not in
+                                           place.person_groups[i]]
+
+                elif place.place_type.name == "CareHome":  # CAREHOME
+                    # Initialise the fixed population of workers
+                    helper.update_place_group(place, group_index=0)
+                    # Initialise a fixed population of residents
+                    person_list = [person for person in cell.persons
+                                   if person.age > 65]
+                    helper.update_place_group(place, group_index=1,
+                                              person_list=person_list)
+
+        # Add temporary population via the update sweep
+        add_temporary_population = UpdatePlaceSweep()
+        add_temporary_population.bind_population(self._population)
+        add_temporary_population(0)
+
+    def create_age_weights(self, place, params):
+        """Function to return a list of people in the correct
+        age range to be added to a place, and the weights
+        for the different age groups.
+
+        Parameters
+        ----------
+
+        place : Place
+            Place to add people to
+        params : dict
+            Dictionary of parameters with age structure data
+
+        Returns
+        -------
+
+        person_list : list
+            List of people who may be in the place
+        weights : list
```

Comment:
```suggestion
        typing.List[Person]
            List of people who may be in the place
        typing.List[float]
```
Don't tend to specify internal variable names in the return for the docs, but we can specify the typing to differentiate between them still